apps/official/components/pages/Faq.tsx

```diff
@@ -87,18 +87,25 @@ export const Faq: React.FC = () => {
         <FaqAccordion title="Was macht die CovMap besonders?">
           <Typography style={{ width: "100%" }}>
             Die CovMap wertet zur regionalen Risikobestimmung neben den offiziellen Fallzahlstatistiken des
-            Robert-Koch-Instituts große Sätze an GPS- und Symptom-Daten aus. Dies erlaubt uns, mit Hilfe eines Modells
-            einen Blick in die Zukunft des Infektionsgeschehens in Deutschland zu werfen und so Risikogebiete schneller
-            zu identifizieren.
-          </Typography>
-        </FaqAccordion>
+            Robert-Koch-Instituts große Sätze an GPS- und Symptom-Daten aus. Diese Daten entstammen aus anderen Apps,
+            wo die Nutzer der Auswertung ihrer GPS Daten zugestimmt haben. Diese Daten sind die Basis für ein von uns
+            entwickeltes Modell, welches den Kontaktindex berechnet. Zudem werden aktuelle Symptome abgefragt, ausgewertet 
+            und fließen zusätzlich in die Risikoeinschätzung mit ein (bald verfügbar). Der Kontaktindex und die 
+            Symptome spiegeln frühzeitig das Infektionsgeschehen wider und ermöglichen so eine zeitnahe Anpassung von
+            Maßnahmen. Im Gegensatz dazu sind die Zahlen der Neuinfektionen, die vom Robert-Koch-Institut veröffentlicht 
+            werden, der Stand der Neuinfektionen von vor circa einer Woche. Dieser Verzug kommt dadurch Zustande, dass
+            eine Person, die neu mit dem Virus infiziert wurde, nach wenigen Tagen Symptome entwickelt (es gibt auch 
+            symptomlose Verläufe), sich erst dann testen lässt, der Test Zeit benötigt und auch die Meldung an das 
+            Gesundeitsamt Zeit in Ansrpuch nimmt. 
```

Comment:
```suggestion
```suggestion
            Maßnahmen. Im Gegensatz dazu stehen die Zahlen der Neuinfektionen, die vom Robert-Koch-Institut veröffentlicht 
            werden. Diese stellen den Stand der Neuinfektionen von vor circa einer Woche dar. Der Verzug kommt Zustande, 
            da eine Person, die neu mit dem Virus infiziert wurde, erst nach einigen Tagen Symptome entwickelt, bzw. 
            sich erst dann testen lässt (es gibt auch symptomlose Verläufe). Zusätzlich benötigt sowohl der Test, als auch 
            die Meldung an das Gesundheitsamt Zeit. 
```
```

Reply:
Da ist irgendwas schief gelaufen mit der suggestion...  das `suggestion` steht da noch mit drin.